ferrocene/tools/pull-subtrees/pull.py

```diff
@@ -1,7 +1,12 @@
-#!/usr/bin/env python3
+#!/usr/bin/env -S uv run
 # SPDX-License-Identifier: MIT OR Apache-2.0
 # SPDX-FileCopyrightText: The Ferrocene Developers
 
+# /// script
+# requires-python = ">=3.12"
+# dependencies = ["requests", "yaml", "pyjwt[crypto]"]
```

Comment:
The dependency is called pyyaml, not yaml.

Reply:
🫠 Only in Python would the package name be different than the relevant import: https://pyyaml.org/wiki/PyYAMLDocumentation